design/account.go

```diff
@@ -171,7 +171,6 @@ var _ = a.Resource("users", func() {
 		a.Description("Send a verification code to the user's email address")
 		a.Response(d.NoContent)
 		a.Response(d.InternalServerError, JSONAPIErrors)
-		a.Response(d.NotFound, JSONAPIErrors)
```

Comment:
When there is unknown identity or user in the token then we should return 401. Not 404. So, I removed 404.